### PR TITLE
Updated Community Ammunition Library

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Added cleaning data and a regexp to match to modules. I'm not sure if the regexp is fine as it is though as it felt dirty to simply end with a or that goes nowhere. 

EDIT: Another note is I don't think putting the merchant module in the fixesGroup is appropriate, it fixes nothing and mostly just adds new traders to rivet city and paradise fall

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -826,17 +826,24 @@ plugins:
       - crc: 0xB362BA90
         util: 'FO3Edit v4.0.3h (Hotfix 1)'
 
-  - name: 'CALIBR.esm'
+
+  - name: '(CALIBR(xMerchant|)\.esp)'
```
Comment (MacSplody) [inline masterlist.yaml]: Removed the extra newline.
You don't need to wrap the filename & extension generally, I can't recall it being used.
`?` can be placed after a capture group, to find if something occurs once or not at all.
`[]` can be used to capture one of any characters listed.
```suggestion
  - name: 'CALIBR(xMerchant)?\.es[mp]'
```

Edit: Cheatsheet for regex. [https://www.rexegg.com/regex-quickstart.html](https://www.rexegg.com/regex-quickstart.html)

**Comment by MacSplody** [comment]
> EDIT: Another note is I don't think putting the merchant module in the fixesGroup is appropriate, it fixes nothing and mostly just adds new traders to rivet city and paradise fall

The group name in app can be different to the anchor.
We could potentially split the group, but that might cause issues for anyone using custom rules with the `Fixes & Resources` group.

```yaml
  - name: &fixesGroup Fixes & Resources
    description: 'A group for modules that fix issues and bugs or add resources for other mods.'
    after: [ *dlcGroup ]
```

**Comment by Superfly-Johnson** [comment]
Fair enough, I commited your suggestion.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**